mongo-docker/start.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Start mongo
+/tmp/entrypoint.sh mongod&
```

Comment:
Nope, nope, and nope. This approach is prettttty hacky and in fact needs to be replaced (before a merge, actually).